src/lib.rs

```diff
@@ -489,61 +388,101 @@ pub trait Rng: RngCore {
         Range::sample_single(low, high, self)
     }
 
-    /// Return a bool with a 1 in n chance of true
+    /// Sample a new value, using the given distribution.
```

Comment:
This commit is just reordering, and this is the diff tool getting confused?

Reply:
Yes, I made sure not to change anything but the order.